esp-hal/CHANGELOG.md

```diff
@@ -59,6 +59,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - ESP32-S2: Made Wi-Fi peripheral non virtual. (#2942)
 - `UartRx::check_for_errors`, `Uart::check_for_rx_errors`, `{Uart, UartRx}::read_buffered_bytes` (#2935)
 - Added `i2c` interrupt API (#2944)
+- Added `with_rx()` and `with_tx()` methods to Uart, UartRx, and UartTx ()
```

Comment:
I think the rebase might have gone awry here

Reply:
Thanks! cleaned this up